tests/integration/app_mock.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
The App mock needs a Docker image that provides an environment to build a scala class. For our existing integration tests (using `unit-integration-tests.sh`) that's easy: Because we've already build such an image (`build-base-image.sh`) to run the integration tests, we can just re-use that one for the app mock. When testing the Mesos containerizer, this doesn't work anymore, because these tests can't run inside a Docker container. Hence we would need to build that image and make it available to the Mesos containerizer, which is a huge effort, especially for something as simple as the app mock. IMO it's easier to just use an easily available Docker image like `python` that makes it very easy to deploy a simple application.